packages/shopify-codemod/test/fixtures/remove-empty-returns/function-declarations.input.js

```diff
@@ -0,0 +1,5 @@
+function whenFinished() {
```

Comment:
Can you add sanity checks for 1) returns that have an argument (not changed), and 2) returns that are nested inside a block (not changed)